test/cypress/pages/ask-cfpb/article.js

```diff
@@ -0,0 +1,10 @@
+export class AskCfpbArticle {
+
```

Comment:
Another minor thing, but we don't really ever call these "articles". Can we update this here and update th filename and references in `ask-cfpb.js` accordingly?

```suggestion
export class AskCfpbAnswerPage {
```

Reply:
Updated --> https://github.com/cfpb/consumerfinance.gov/pull/6206/commits/808b6b10cbf5d6bccd3993cba47d8b3309615f69